client/components/Editor/index.jsx

```diff
@@ -20,36 +20,35 @@ class Editor extends Component {
     constructor(props) {
         super(props);
 
-        if (!this.props.match.params.id) {
-            this.props.match.params.id = 'new';
```

Comment:
Changing the URL params data without changing the actual URL felt like a potential source of confusion in the future.

Reply:
I think this was a vestigial change that was no longer needed after you made the `<NewScenario>` wrapper.

Reply:
I actually found this logic helpful for figuring out the tabs we need later in the constructor and for setting state, so I think some form of it is worth keeping for now.